app/views/shared/nav/_nav_links.html.haml

```diff
@@ -2,6 +2,11 @@
   - if responsive?
     -# .hidden-with-nav is hidden > 979px
     %li.hidden-with-nav= link_to t('pages.cart') + " (#{current_cart.order_details.count})", :cart
+    %li{class: [home_button.tab_class(controller), "useButton"], id: home_button.tab_id }  
+      = home_button.to_html
+    - if SettingsHelper.feature_on?(:use_manage)
```

Comment:
I believe the extra "Manage Facilities" link is coming from the `navigation_links` list.